test/tests.py

```diff
@@ -0,0 +1,105 @@
+#                 0.5).create_column_embeddings().get_metadata()
+#         # to_hash = ""
+#         # m = hashlib.sha256()
+#         # for name, valee in metadata.items():
+#         #     m.update(valee.hash())
+#         #     to_hash = f"{to_hash}|{name}{valee.hash()}"
+#         # # print(f"hashed: {hash(to_hash)} , {hash(to_hash)}")
+#         # # print(f"not_hashed: {to_hash}")
+#         # print(f"hash from hashlib: {m.hexdigest()}")
+#         return metadata
+#
+#     def test_cross_compare(self):
+#         metadata = self.create_metadata()
+#
+#         comparator = ComparatorForDatasets(metadata) # compare
+#         res = comparator.cross_compare()
+#         corr1 = self.print_correlation(res)
+#
+#         comparator2 = ComparatorForDatasets(metadata)  # compare
+#         res2 = comparator2.cross_compare()
+#         corr2 = self.print_correlation(res2)
+#
+#         self.assertEqual(corr1, corr2)
+#
+#         print("Column names")
+#         res = comparator.cross_compare_column_names()
+#         corr1 = self.print_correlation(res)
+#
+#
+#
+#     #def test_similarity_of_metadata(self):
+#         # database, names = load__csv_files_from_folder("../data")
+#         # metadata: dict[str, DataFrameMetadata] = defaultdict()
+#         # for dataframe, name in zip(database, names):
+#         #     metadata[name] = DataFrameWithStat(dataframe).compute_correlation(
+#         #         0.5).create_column_embeddings().get_metadata()
+#         #     icompl = metadata[name].column_incomplete
+#         #     categ = metadata[name].column_categorical
+#         #
+#         # metadata2: dict[str, DataFrameMetadata] = defaultdict()
+#         # for dataframe, name in zip(database, names):
+#         #     metadata2[name] = DataFrameWithStat(dataframe).compute_correlation(
+#         #         0.5).create_column_embeddings().get_metadata()
+#         #     icompl = metadata[name].column_incomplete
+#         #     categ = metadata[name].column_categorical
+#         # # self.assertEqual(metadata, metadata2)
+#         # for m1, m2 in zip(metadata2.values(), metadata.values()):
+#         #     self.assertEqual(m1, m2)
+#         #
+#         # database2, names2 = load__csv_files_from_folder("../data")
+#         # metadata3: dict[str, DataFrameMetadata] = defaultdict()
+#         # for dataframe, name in zip(database2, names2):
+#         #     metadata3[name] = DataFrameWithStat(dataframe).compute_correlation(
+#         #         0.5).create_column_embeddings().get_metadata()
+#         #     icompl = metadata[name].column_incomplete
+#         #     categ = metadata[name].column_categorical
+#         #
+#         # # self.assertEqual(metadata, metadata3)
+#         # # self.assertEqual(metadata2, metadata3)
+#         # for m1, m2 in zip(metadata3.values(), metadata.values()):
+#         #     self.assertEqual(m1, m2)
+#
+#
+#
+#
+#
+#
+# if __name__ == '__main__':
+#     unittest.main()
```

Comment:
Wouldn't be enough to comment this?

Reply:
Or use `pytest.ini` to ignore certain tests.

Reply:
Fixed